pkg/cluster/nodes/node.go

```diff
@@ -319,3 +319,59 @@ func (n *Node) WriteKubeConfig(dest string, hostPort int) error {
 
 	return ioutil.WriteFile(dest, buff.Bytes(), 0600)
 }
+
+// WriteFile writes temporary file on the host
+// and copies it to the node
+func (n *Node) WriteFile(dest, content string) error {
```

Comment:
didn't we have a method for this already?

Reply:
no as far as I can see. I'm planning to use this to refactor a couple of places in a separate PR.